src/platform/ESP32/Logging.cpp

```diff
@@ -29,28 +29,28 @@ void LogV(const char * module, uint8_t category, const char * msg, va_list v)
     {
     case kLogCategory_Error: {
         {
-            esp_log_write(ESP_LOG_ERROR, tag, LOG_COLOR_E "E (%" PRIu32 ") %s: ", esp_log_timestamp(), tag);
+            printf(LOG_COLOR_E "E (%" PRIu32 ") %s: ", esp_log_timestamp(), tag);
```

Comment:
Maybe add some comment why printf is uded here, so it is clear that esp_log will break logging.

Reply:
Thanks @arkq , makes sense, added the comment.